web/app.js

```diff
@@ -1769,6 +1773,27 @@ function webViewerSidebarViewChanged(evt) {
   }
 }
 
+function setupHashUpdating() {
+  let tid = false;
+
+  // set up the update function
+  let updatePos = () => {
+    return PDFViewerApplication.pdfHistory.pushCurrentPosition();
```

Comment:
I don't think `pushCurrentPosition` returns anything, so I think you can simplify by removing `updatePos` altogether and just calling this method in `setTimeout` below.